adrs/php/0002_module_structure.md

```diff
@@ -0,0 +1,29 @@
+# Module structure
+
+## Motivation
+
+Standardize the design of the different modules. Considering a module as an internal package that contains
```

Comment:
This ADR standardizes the design of different modules. A module divides this project into smaller parts. Each module defines a set of classes that belong together and are independently from classes in other modules.

Does this make sense?